clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -815,8 +815,8 @@ def err_unexpected_template_in_destructor_name : Error<
   "'template' keyword not permitted in destructor name">;
 def err_unexpected_template_after_using : Error<
   "'template' keyword not permitted after 'using' keyword">;
-def err_two_right_angle_brackets_need_space : Error<
-  "a space is required between consecutive right angle brackets (use '> >')">;
+def err_two_right_angle_brackets_need_space : ExtWarn<
```

Comment:
nit: I think `ext_` is the more common diag prefix for `ExtWarn`

Reply:
Yes, if we accept this, the name of the diagnostic has to change to be consistent.